### PR TITLE
Added image connectivity check

### PR DESCRIPTION
**Pull request by RoyUP9**

### CODE REVIEW

File: cli/cmd/checkRunner.go [hunk 1]
```diff
@@ -315,3 +322,104 @@ func checkPermissionExist(group string, resource string, verb string, exist bool
...
+	logger.Log.Infof("%v docker hub image pulled", fmt.Sprintf(uiUtils.Green, "√"))
+	return true
+}
+
+func checkImagePulled(ctx context.Context, kubernetesProvider *kubernetes.Provider, podName string) error {
+	podExactRegex := regexp.MustCompile(fmt.Sprintf("^%s$", podName))
+	podWatchHelper := kubernetes.NewPodWatchHelper(kubernetesProvider, podExactRegex)
+	eventChan, errorChan := kubernetes.FilteredWatch(ctx, podWatchHelper, []string{config.Config.MizuResourcesNamespace}, podWatchHelper)
+
+	timeAfter := time.After(30 * time.Second)
```
Comment (RoyUP9) [inline cli/cmd/checkRunner.go:351]: should i extract 30 to config or something like that?

File: cli/cmd/checkRunner.go [hunk 2]
```diff
@@ -315,3 +322,104 @@ func checkPermissionExist(group string, resource string, verb string, exist bool
...
+		logger.Log.Errorf("%v error while creating images connectivity resources, err: %v", fmt.Sprintf(uiUtils.Red, "✗"), err)
+		return false
+	}
+
+	if err := checkImagePulled(ctx, kubernetesProvider, podName); err != nil {
+		logger.Log.Errorf("%v docker hub image not pulled, err: %v", fmt.Sprintf(uiUtils.Red, "✗"), err)
+		return false
+	}
+
+	logger.Log.Infof("%v docker hub image pulled", fmt.Sprintf(uiUtils.Green, "√"))
```
Comment (IgorGov) [inline cli/cmd/checkRunner.go:342]: cluster is able to pull mizu containers from docker hub

File: cli/cmd/checkRunner.go [hunk 3]
```diff
@@ -315,3 +322,104 @@ func checkPermissionExist(group string, resource string, verb string, exist bool
...
+	podName := "images-connectivity"
+
+	defer removeImagesConnectivityResources(ctx, kubernetesProvider, podName)
+	if err := createImagesConnectivityResources(ctx, kubernetesProvider, podName); err != nil {
+		logger.Log.Errorf("%v error while creating images connectivity resources, err: %v", fmt.Sprintf(uiUtils.Red, "✗"), err)
+		return false
+	}
+
+	if err := checkImagePulled(ctx, kubernetesProvider, podName); err != nil {
+		logger.Log.Errorf("%v docker hub image not pulled, err: %v", fmt.Sprintf(uiUtils.Red, "✗"), err)
```
Comment (IgorGov) [inline cli/cmd/checkRunner.go:338]: cluster is not able to pull mizu containers from docker hub

File: cli/cmd/checkRunner.go [hunk 4]
```diff
@@ -315,3 +322,104 @@ func checkPermissionExist(group string, resource string, verb string, exist bool
 	logger.Log.Infof("%v can %v %v in group '%v'", fmt.Sprintf(uiUtils.Green, "√"), verb, resource, group)
 	return true
 }
+
+func checkImagesConnectivity(ctx context.Context, kubernetesProvider *kubernetes.Provider) bool {
+	logger.Log.Infof("\nimages-connectivity\n--------------------")
```
Comment (IgorGov) [inline cli/cmd/checkRunner.go:327]: image pull in cluster

**Pull request merged**